src/Gateways/PorticoConnector.ts

```diff
@@ -1324,4 +1337,13 @@ export class PorticoConnector extends XmlGateway implements IPaymentGateway {
 
     return result;
   }
+
+  protected hydrateFoundTransactions(root: Element): Transaction {
```

Comment:
@wdesimini which fields do we need out of this request? Here are all available: https://cert.api2.heartlandportico.com/Gateway/PorticoSOAPSchema/build/Default/webframe.html#Portico_xsd~e-PosResponse~e-Ver1.0~e-Transaction~e-FindTransactions.html

@lundhjem

Reply:
I think we're mainly looking for the transaction amount (looks like either Amt or AuthAmt) to stick into the amount parameter of the reversal request, but maybe just return the whole transaction?

@lundhjem could you confirm/provide more info?

Reply:
here is all the data these are returning: 

![Screen Shot 2020-06-05 at 10 37 00 AM](https://user-images.githubusercontent.com/3927506/83896093-3ac17780-a719-11ea-9829-257eb17bb767.png)
![Screen Shot 2020-06-05 at 10 36 53 AM](https://user-images.githubusercontent.com/3927506/83896101-3d23d180-a719-11ea-96c7-270281302d98.png)

Reply:
@darrenjaworski Noice!! Thanks for sharing